### PR TITLE
feat/bruce-banner

### PR DESCRIPTION
**Pull request by rossanodan**
## About this pull request

- [x] Removed documentation folder and scripts - too soon to write something useful
- [x] Removed Semantic-UI
- [x] Removed all the components so that we can resume working with a clean codebase
- [x] Updated react-scripts
- [x] Added [Tailwind CSS](https://tailwindcss.com/)
- [x] Added ESLint script `npm run lint`
- [x] Implemented pre-commit hook with [husky](https://github.com/typicode/husky)
- [x] Updated the README.md
- [x] Started updating the CONTRIBUTING.md with a more detailed induction to the project
- [x] Added CODEOWNERS so that every change has to be reviewed and approved by the code owner to be merged
- [x] Set `jest` for unit testing (`npm run test`)
- [x] Set up GitHub branches protection to require the review by code owners 

Screenshot 2020-09-20 at 17 49 43
This should be enough but I don't see any required review in the reviewers section of this pull request. @eddiejaoude any idea about this?

---

Since we don't have any IDs or any reference for our branches, I generated the branch name using [this tool](https://sprintnamegenerator.com/). It looks funny! 😄

### CODE REVIEW

**Comment by eddiejaoude** [comment]
> This should be enough but I don't see any required review in the reviewers section of this pull request. @eddiejaoude any idea about this?

Sorry I understand your message when you said `code owners`. This shouldn't be checked, not sure why it was, it is not on any other branch or project in our community - I have now removed it. All our PRs need 1 review from anyone, that should be enough.

> Since we don't have any IDs or any reference for our branches

With branches in our other repos we use `<username>-issue-<number>`, so in my case `eddiejaoude-issue-123` with commit messages like conventional change log style `feat(module): description .... <issue-number>`. I should document this somewhere central and sync it with a GitHub Action to all our repos 

Btw when merging, we do squash when going to `develop` if the PR has lots of repeated commit messages

File: CODEOWNERS
Comment (eddiejaoude) [inline CODEOWNERS:5]: Is this file needed?

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: CODEOWNERS
Comment (rossanodan) [reply CODEOWNERS:5]: With the admin approval thing, I don't think so. But I added it to have this

Screenshot 2020-09-21 at 00 02 14

It's you now because you reviewed it. But it'll be me in case I will approve someone else's PR.

Comment (eddiejaoude) [reply CODEOWNERS:5]: ok thanks 👍 - it can go in a new PR, so we can get this merged

**Pull request merged**
